### DISCUSSION TITLE
Citizen Science ideas

### DISCUSSION THREAD

**Discussion by scollis** [CROCUS-Urban/measurement-strategy#4] [General]
While there were initial ideas for citizen science in the proposal we will be adaptable and looking to ideas from the CROCUS teama and the community at large. One thing that was made clear in the events surrounding the announcement is there will be a great appetite for this kind of activity and there are educational institutions (k-12 and higher ed) keen to engage

**Comment by swnesbitt** [comment]
One idea would be to develop a tool (phone app) that could document and collect data about flooding.  This could be extended to more quantitative data collection (depth of water, streamflow), but for now, just a placeholder.

**Reply by scollis** [reply]
Agree! And one can also use a Twitter hashtag... eg CROCUSFloods and use the twitter API.. We would need to get HSR sign off for that one.. no big deal.
